lib/master.js

```diff
@@ -58,46 +65,14 @@ module.exports = function master() {
           thread.kill();
         })
         .on('exit', function () {
-          var totals;
           complete = complete + 1;
           log(`Thread exited. ${complete}/${this.instances.length}`);
           thread.removeAllListeners('message');
           thread.removeAllListeners('error');
           thread.removeAllListeners('exit');
           if (complete === this.instances.length) {
-            totals = {total: 0, pass: 0, fail: 0};
-            log('Everything done, shutting down the thread pool.');
-            results.forEach(function (result) {
-              totals.total = totals.total + result.total;
-              totals.pass = totals.pass + result.pass;
-              totals.fail = totals.fail + result.fail;
-            });
-
-            let summary = JSON.stringify({
-              iterations: results,
-              summary: totals
-            }, null, '\t')
-            /* eslint-disable */
-            // console.log(summary);
-            let tabl = new table({
-              head: ['tags', 'pass', 'fail', 'total', 'report']
-            });
-            results.forEach(function (iter) {
-              let tags = '';
-              Object.keys(iter.tags).forEach(function (key) {
-                if (key === 'uid') {
-                  return;
-                }
-                tags += `${key}: ${iter.tags[key]}\n`
-              });
-              tabl.push([tags, iter.pass, iter.fail, iter.total, iter.reportFile || '']);
-            });
-            tabl.push(['TOTALS', totals.pass, totals.fail, totals.total]);
-            console.log(tabl.toString());
-            if (this.config.get('output:reports')) {
-              fs.writeFileSync(`${this.config.get('output:reports')}/summary.json`, summary)
-            }
-            /* eslint-enable */
+            console.log(instanceResults);
```

Comment:
I think it's possible that when we run parallel by file this will log a lot even though no tests are being run.